kai/data/misc/jonah-notes.md

```diff
@@ -0,0 +1,24 @@
+# Jonah Notes
```

Comment:
@JonahSussman do you want to commit this?

I'm not against committing notes/questions/etc to help show our thinking if this is what you want.

Just wanted to check if this was included by accident

Reply:
Yep, was committed in error. Removed